omod/src/main/webapp/fragments/patientflagsDashboard.gsp

```diff
@@ -15,7 +15,7 @@
                 if (!response) {
                     ${ ui.message("coreapps.none ") }
                 } else {
-                    jq("#flags").html(response.replace("{=", "").replace("}", ""));
```

Comment:
@wikumChamith does this work on lower platform versions?

Reply:
I tested this on Reference Application 2.11.0 but this is not working there.

Reply:
@wikumChamith meaning that this change breaks the module for those who are running lower platform versions.